pkg/content/zipper.go

```diff
@@ -0,0 +1,133 @@
+package content
```

Comment:
it would be nice to explain why extra zipper/archiver is needed here and why mholt cannot be used here.